[Edit] Luau (#6624)
Codecademy/docs: content/luau/luau.md

---
Title: 'Luau'
Description: 'Luau is a scripting language for Roblox, based on Lua 5.1 and backwards compatible, allowing Lua 5.1 code to run within the Luau environment.'
Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/luau'
CatalogContent:
  - 'paths/computer-science'
---

> **Luau** is a scripting language derived from [Lua](https://www.codecademy.com/resources/docs/lua). It was created specifically for [Roblox](https://www.roblox.com/), an online game platform and game creation system. Luau is backwards compatible with Lua 5.1, meaning that Lua 5.1 code can be executed within the Luau environment. It is used by Roblox game developers to write game logic, and by Roblox engineers to implement large parts of the user-facing application code as well as plugins for Roblox Studio.

Reviewer: ```suggestion
**Luau** is a scripting language derived from [Lua](https://www.codecademy.com/resources/docs/lua). It was created specifically for [Roblox](https://www.roblox.com/), an online game platform and game creation system. Luau is backwards compatible with Lua 5.1, meaning that Lua 5.1 code can be executed within the Luau environment. Roblox game developers use it to write game logic and implement large parts of the user-facing application code and plugins for Roblox Studio.
```
